jobs/resource_job.go

```diff
@@ -74,6 +75,17 @@ type TaskDependency struct {
 	TaskKey string `json:"task_key,omitempty"`
 }
 
+// BEGIN Jobs + Repo integration preview
+type GitSource struct {
+	Url      string `json:"git_url" tf:"alias:url"`
+	Provider string `json:"git_provider" tf:"alias:provider"`
```

Comment:
`provider` is an illegal property name for terraform, because it's a reserved keyword...

@alexott maybe we should keep the non-aliased properties after all, to simplify the support in the long run?

Reply:
not sure. I really don't like repetition of the names, because we already in the `git` block

Reply:
think about difficulties of our tech support troubleshooting the configs ;)